docs/public/presets/toml/no-runtime-versions.toml

```diff
@@ -2,14 +2,20 @@
 format = "via [$symbol]($style)"
 
 [buf]
-format = "via [$symbol]($style)"
+format = "with [$symbol]($style)"
+
+[c]
+format = "via [$symbol($name)]($style)",
```

Comment:
Some of the TOML syntax is invalid

Reply:
Oops, I knew this would happen /o\. Fixed now.